test/test_clicommand.py

```diff
@@ -132,3 +133,23 @@ def test_bad_range_options(self):
                            "profilename", "hosts",
                            "a:d:b:s", "--auths",
                            "auth_1", "auth2"])
+
+    def test_a_auth_add(self):
```

Comment:
I tried using http://pytest-ordering.readthedocs.io/en/develop/#other-markers to order the tests, but that didn't seem to work for me. It runs tests in alphabetical order currently. These prefixes control when the test are getting run. I essentially run the add auth and add profile commands before all others and then clear all auth and profile as the last tests.